gas-mask-se.user.js

```diff
@@ -14,30 +14,34 @@
 // @version       1.3
 // ==/UserScript==
 
-if(true || location.search.indexOf("smokeypost=true") !== -1){
+if (true || location.search.indexOf("smokeypost=true") !== -1) { // eslint-disable-line no-constant-condition
```

Comment:
Why don't you introduce a NOTFOUND const so yo can do without the eslint supression?